src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -443,14 +443,14 @@ class LedgerMasterImp
             {
                 ApplyFlags tepFlags = tapNONE;
 
-                if (app_.getHashRouter ().addSuppressionFlags (
-                        it.first.getTXID (), SF_SIGGOOD))
-                    tepFlags = static_cast<ApplyFlags> (
-                        tepFlags | tapNO_CHECK_SIGN);
```

Comment:
I'm leaving this instance out because when the Transactor calls checkValidity, the same flags will get checked, so this instance is redundant.